PokemonSearch/PokemonSearch/SearchTableViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SearchTableViewController.swift
+//  PokemonSearch
+//
+//  Created by Jocelyn Stuart on 1/25/19.
+//  Copyright © 2019 JS. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableViewController: UITableViewController, UISearchBarDelegate {
+
+    var pokemonController = PokemonController()
+    
+    var pokemon: Pokemon?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        guard let searchTerm = searchBar.text, !searchTerm.isEmpty else { return }
+        
+        pokemonController.searchForPokemon(with: searchTerm.lowercased()) { (pokemon, error) in
+            DispatchQueue.main.async {
+                self.pokemon = pokemon
+                self.tableView.reloadData()
+            }
+        }
+    }
+    
+    // MARK: - Table view data source
+
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return pokemons.count
+    }
+
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "InfoCell", for: indexPath) as! SearchTableViewCell
+
+        let pokemon = self.pokemons[indexPath.row]
```

Comment:
Same as here. You need to unwrap pokemon object and use it as you data source
```suggestion
        guard let pokemon = pokemon else { return cell }
```